ui.tests/test-module/specs/actions/submit/submit.runtime.spec.js

```diff
@@ -141,4 +141,11 @@ describe("Form with Submit Button", () => {
         // To Fix: this is failing the test saying waiting for page load
         //cy.wait('@redirected');
     })
-})
+
+    it("Submit Action test without passing any custom submit event", () => {
+        cy.previewForm(basicSubmitPagePath);
+        cy.get(`.cmp-adaptiveform-button__widget`).click().then(x => {
```

Comment:
Please write test case for submit success handler, this test already exists.

Reply:
Thankyou message comes only after submit success handler is executed https://github.com/adobe/aem-core-forms-components/blob/f9d0f0d11ca156b48c7b58da7b45ab3174ecd123/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js#L39 
I have added event listener for`submitStart` event which is triggered after the submitSuccess event